src/app/views/profile/trophies/game/game.vue

```diff
@@ -1,55 +1,36 @@
 <template>
 	<div>
-		<template v-if="completion">
-			<app-trophy-completion
-				class="-completion"
-				:total="completion.totalCount"
-				:achieved="completion.achievedCount"
-				:experience="completion.experience"
-			/>
-		</template>
-		<template v-if="game">
-			<div class="-game-links">
-				<div>
-					<router-link :to="game.routeLocation">
-						<app-game-thumbnail-img class="-game-thumbnail" :game="game" />
-					</router-link>
-				</div>
-				<div class="-game-links-nav">
-					<router-link :to="game.routeLocation">
-						<translate>View game</translate>
-					</router-link>
-					<br />
-					<router-link
+		<div class="row" v-if="game">
+			<div v-if="completion" class="col-sm-7">
+				<app-trophy-completion
+					class="-completion"
+					:total="completion.totalCount"
+					:achieved="completion.achievedCount"
+					:experience="completion.experience"
+				/>
+				<p>
```

Comment:
I think these buttons should show even if you are looking at someone else's profile.

Reply:
Yeah, I agree, this is a bug then